counsel.el

```diff
@@ -4050,14 +4051,17 @@ current outline heading in markdown-mode buffers. See
 
 (defun counsel-outline-title-latex ()
   "Function used by `counsel-outline' to get the title of the
-current outline heading in latex-mode buffers. See
-`counsel-outline-settings'."
-  ;; `outline-regexp' is set by `latex-mode' (see
-  ;; `LaTeX-outline-regexp') to match section macros, in which case we
-  ;; get the section name, as well as `\appendix', `\documentclass',
-  ;; `\begin{document}' and `\end{document}', in which case we simply
-  ;; return that.
-  (if (and (assoc (match-string 1) LaTeX-section-list) ;; section macro
+current outline heading in latex-mode buffers (built-in mode or
+AUCTeX package). See `counsel-outline-settings'."
+  ;; `outline-regexp' is set by `latex-mode' (see variable
+  ;; `latex-section-alist' for the built-in mode or function
+  ;; `LaTeX-outline-regexp' for the AUCTeX package) to match section
+  ;; macros, in which case we get the section name, as well as
+  ;; `\appendix', `\documentclass', `\begin{document}' and
+  ;; `\end{document}', in which case we simply return that.
+  (if (and (assoc (match-string 1) (if (boundp 'LaTeX-section-list)
+                                       LaTeX-section-list
+                                     latex-section-alist)) ; section macro
```

Comment:
How about:

```el
(or (bound-and-true-p LaTeX-section-list)
    (bound-and-true-p latex-section-alist)))
```

This removes the need for `(defvar LaTeX-section-list)` or `(defvar latex-section-alist)` and makes the function a tiny bit less mode/package-dependent (not that this is particularly important).

Reply:
Thanks, done